imagetest/test_suites/storage_perf/startupscripts/install_fio.sh

```diff
@@ -0,0 +1,16 @@
+if [[ -f /usr/bin/apt ]]; then
```

Comment:
You won't need sudo in this case (startup scripts run as root).

Reply:
done